vars/pod.groovy

```diff
@@ -27,6 +27,9 @@ def call(params = [:], Closure body) {
     }
     if (params['cpu']) {
         podObj['spec']['containers'][0]['resources']['requests']['cpu'] = params['cpu'].toString()
+        // Also propagate CPU count to MAKE_JOBS, because it can be hard in a Kubernetes environment
+        // to determine how much CPU one should really use.
+        podObj['spec']['containers'][0]['env'] += ['name': 'MAKE_JOBS', 'value': params['cpu'].toString()]
```

Comment:
WDYT about `NCPUS` instead to make it more builder agnostic?

Reply:
Sure, done.